src/SlackThreads.jl

```diff
@@ -72,7 +72,8 @@ end
 
 """
     (thread::SlackThread)(text::AbstractString, uploads...;
-                          format_message_counts=format_message_counts)
+                          format_message_counts=format_message_counts,
+                          options...)
```

Comment:
By doing this as a keyword argument to `thread`:

1. we can't make an internal keywords (like that `format_message_counts`) that have the same name as a Slack API option for `chat.postMessage`. I think that's fine though; pretty easy to avoid since there's like 10 options: https://api.slack.com/methods/chat.postMessage#args.
2. Also, if Slack creates an API key that isn't a typical Julia identifier (e.g. if they named a key with a space) then it wouldn't be ergonomic to call this way. Though technically, you still can with the `var` string macro:
    ```julia
    julia> f(; kwargs...) = @show kwargs
    f (generic function with 1 method)

    julia> f(; var"hi bye"=1)
    kwargs = Base.Pairs(Symbol("hi bye") => 1)
    pairs(::NamedTuple) with 1 entry:
      Symbol("hi bye") => 1
    ```

I think those limitations are mostly theoretical and outweighed by the simplicity of doing `thread("my message"; link_names=true)`, for example.